homeassistant/components/honeywell/climate.py

```diff
@@ -35,47 +34,19 @@
 )
 from homeassistant.const import (
     ATTR_TEMPERATURE,
-    CONF_PASSWORD,
-    CONF_REGION,
-    CONF_USERNAME,
+    PERCENTAGE,
     TEMP_CELSIUS,
     TEMP_FAHRENHEIT,
 )
-import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_COOL_AWAY_TEMPERATURE, CONF_HEAT_AWAY_TEMPERATURE, DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_FAN_ACTION = "fan_action"
 
-CONF_COOL_AWAY_TEMPERATURE = "away_cool_temperature"
-CONF_HEAT_AWAY_TEMPERATURE = "away_heat_temperature"
-CONF_DEV_ID = "thermostat"
-CONF_LOC_ID = "location"
-
-DEFAULT_COOL_AWAY_TEMPERATURE = 88
-DEFAULT_HEAT_AWAY_TEMPERATURE = 61
-
 ATTR_PERMANENT_HOLD = "permanent_hold"
 
-PLATFORM_SCHEMA = vol.All(
```

Comment:
It is fine to remove the platform configuration, however, you have to make sure existing configurations of users are imported into a configuration entry. That way, we migrate users instead of leaving them with a broken configuration.